### PR TITLE
core/rawdb: db inspect move 'config' and 'shutdown' into 'meta data'

### PR DESCRIPTION
**Pull request by Evolution404**
Avoid the error message of Database contains `unaccounted data`
image

### CODE REVIEW

File: core/rawdb/database.go [hunk 1]
```diff
@@ -369,8 +370,6 @@ func InspectDatabase(db ethdb.Database, keyPrefix, keyStart []byte) error {
 			bytes.HasPrefix(key, []byte("bltIndex-")) ||
 			bytes.HasPrefix(key, []byte("bltRoot-")): // Bloomtrie sub
 			bloomTrieNodes.Add(size)
-		case bytes.Equal(key, uncleanShutdownKey):
-			shutdownInfo.Add(size)
```
Comment (rjl493456442) [inline core/rawdb/database.go:373]: Please also count it in the meta.

Comment (Evolution404) [reply core/rawdb/database.go:373]: It has been counted in the `default` 😂

**Review by rjl493456442 — APPROVED** [review]
LGTM, thanks for catching it!

**Comment by holiman** [comment]
LGTM, but the linter complains: 
```
core/rawdb/database.go:319: File is not `goimports`-ed (goimports)
		metadata     stat
		unaccounted  stat
```
It needs a `goimports`, (depending on your tooling, may be `go tool goimports`)

File: core/rawdb/database.go [hunk 2]
```diff
@@ -316,9 +316,8 @@ func InspectDatabase(db ethdb.Database, keyPrefix, keyStart []byte) error {
 		bloomTrieNodes stat
 
 		// Meta- and unaccounted data
-		metadata     stat
-		unaccounted  stat
-		shutdownInfo stat
+		metadata    stat
+		unaccounted stat
```
Comment (Evolution404) [inline core/rawdb/database.go:320]: But I didn't modify these two line, how would this happen?😂

**Pull request merged**